gallery/src/components/NxDrawer/NxDrawerConditionalRenderingExample.tsx

```diff
@@ -0,0 +1,124 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React from 'react';
+
+import { NxDrawer,
+  NxBackButton,
+  NxButton,
+  NxH1,
+  NxP,
+  NxPageMain,
+  NxPageTitle,
+  useToggle
+} from '@sonatype/react-shared-components';
+
+export default function NxDrawerConditionalRenderingExample() {
+  const [mountDrawer, toggleMountDrawer] = useToggle(false);
+  const [open, toggleOpen, setOpen] = useToggle(true);
```

Comment:
I don't think `useToggle` is ideal for either of these.  Everywhere in this file that a `toggle...` function is being called, you're really trying to set the value a specific way and relying on the assumption that it is already the other way.  Just use `useState` and the setters that take explicit values

Reply:
> I don't think `useToggle` is ideal for either of these. Everywhere in this file that a `toggle...` function is being called, you're really trying to set the value a specific way and relying on the assumption that it is already the other way. Just use `useState` and the setters that take explicit values

Fixed: 6838e664928dc1e9fe633d1c16a0db97fdc5cad6